src/physics/em/LivermorePEParams.cc

```diff
@@ -183,12 +184,7 @@ LivermorePEParams::extend_shells(const ElementInput& inp)
 Span<real_type>
 LivermorePEParams::extend_data(const std::vector<real_type>& data)
 {
-    CELER_EXPECT(host_data_.size() + data.size() <= host_data_.capacity());
-
-    // Allocate data
-    host_data_.insert(host_data_.end(), data.begin(), data.end());
-    return Span<real_type>{host_data_.data() + host_data_.size() - data.size(),
-                           data.size()};
+    return celeritas::extend(data, &host_data_);
```

Comment:
I thought I'd leave it in place since the function has a little extra error checking and embeds the `host_data_` -- replacing it causes many one-line statements to wrap to two 😅

Reply:
Makes sense, thanks!